.gitignore

```diff
@@ -1,36 +1,57 @@
+# Quick format and style primer:
+#
+#  * If a pattern is meant only for a specific location, it should have a
+#    leading slash, like `/staticfiles.json`.
+#    * In principle any non-trailing slash (like `zproject/dev-secrets.conf`)
+#      will do, but this makes a confusing pattern.  Adding a leading slash
+#      is clearer.
+#
+#  * Patterns like `.vscode/` without slashes, or with only a trailing slash,
+#    match in any subdirectory.
+#
+#  * Subdirectories with several internal things to ignore get their own
+#    `.gitignore` files.
+#
+# See `git help ignore` for details on the format.
+
+## Config files for the dev environment
+/zproject/dev-secrets.conf
+/tools/conf.ini
+/tools/custom_provision
+
+## Byproducts of setting up and using the dev environment
 *.pyc
-*~
+
+/queue_error
```

Comment:
I think this one is no longer needed; we moved these to `var/log/queue_error/` as a side effect of moving the logs.